en/option/component/data-zoom-slider.md

```diff
@@ -213,6 +213,18 @@ Whether to update view while dragging. If it is set as `false`, the view will be
     dataZoomName = 'dataZoom-slider'
 ) }}
 
+## width(string|number) = 30
+
+<ExampleUIControlNumber default="30"/>
+
+Width of dataZoom-slider component. Default to 30 when vertical, adaptive when horizontal.
```

Comment:
Where is the source code of `30`? It seems the default value is `'ph'` where the layout is calculated automatically. You should also mention what happens when all `left`, `right` and `width` are defined, which one is overriden.

Reply:
I saw the source code use `DEFAULT_FILLER_SIZE` and its value is 30 😂
https://github.com/apache/echarts/blob/4a8a7f58e772586eebfbb080d2d14f5ded5478c0/src/component/dataZoom/SliderZoomView.ts#L231

Reply:
Hi @Ovilia , is there any other problem? 😂